atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -408,6 +409,9 @@ def add_log_type(output):
         output_files = [add_log_type(add_buildroot_id(md, buildroot_id))
                         for md in osbs_logs.get_log_files(self.osbs, self.build_id)]
 
+        if pnc_build_ids:
```

Comment:
would be nice to have this in '_update_extra' in KojiImportPlugin

Reply:
Moved to _updated_extra